deployment/docker-dev/REQUIREMENTS.txt

```diff
@@ -25,3 +25,5 @@ beautifulsoup4==4.5.1
 django-tinymce==2.6
 django-filebrowser==3.7.2
 django-tinymce-filebrowser==0.2.1
+urllib3==1.22
```

Comment:
Added urllib3 so we can view Celery tasks errors.

Reply:
My first time to find `urllib3`.